source/network/ODServer.cpp

```diff
@@ -1507,20 +1507,20 @@ bool ODServer::processClientNotifications(ODSocketClient* clientSocket)
                         // We look for the Skirmish or multiplayer prefix and keep it.
                         uint32_t indexSk = fileLevel.find(SAVEGAME_SKIRMISH_PREFIX);
                         uint32_t indexMp = fileLevel.find(SAVEGAME_MULTIPLAYER_PREFIX);
-                        if((indexSk != std::string::npos) && (indexMp == std::string::npos))
+                        if(((size_t)indexSk != std::string::npos) && ((size_t)indexMp == std::string::npos))
```

Comment:
I guess it would have been better to define indexSk as a size_t instead of casting every time there is a compare

Reply:
Actually, it seems like Travis build fails because of that. I'm fixing it

Reply:
Yes I pushed a change too late few minutes after you merged :-) but feel free.

Reply:
I already created the PR, thanks ^^